modules/openapi-generator/src/main/resources/dart-jaguar/class.mustache

```diff
@@ -2,6 +2,7 @@ part '{{classFilename}}.jser.dart';
 
 class {{classname}} {
   {{#vars}}{{#description}} /* {{{description}}} */{{/description}}
+  @Alias('{{baseName}}')
```

Comment:
@jaumard thanks for the PR. I would suggest using `{{{baseName}}}` as `{{baseName}}` may escape special characters in HTML.

Reply:
But here we want them to be escaped anyway. It's should be plain text only as it's the name of the JSON field

Reply:
No problem :) it's done ^^